runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -81,6 +81,8 @@ void deactivateDevice()
 
 extern "C" {
 
+void __catalyst__host__rt__unrecoverable_error() { RT_FAIL("Unrecoverable error."); }
```

Comment:
Would you add a simple test for this new method in `Test_LightningCoreQIS.cpp` for the codecov report?

Reply:
https://github.com/PennyLaneAI/catalyst/pull/447/commits/372f237f8f9c38e548794aaf40bce1d286c86817